sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 ## 5.2.0-preview.4 (Unreleased)
 
+### Acknowledgments
+
+Thank you to our developer community members who helped to make the Event Hubs client libraries better with their contributions to this release:
+
+- Daniel Marbach _([GitHub](https://github.com/danielmarbach))_
+
+### Changes
+
+#### Key Bug Fixes
+
+- The `EventProcessorClient` will now perform an eager validation of connection strings upon creation.  Previously, validation was deferred until a partition was claimed which made debugging difficult.
+
+- Fixed an issue where failure to create an AMQP link would lead to an AMQP session not being explicitly closed, causing connections to the Event Hubs service to remain open until a garbage collection pass was performed.
```

Comment:
Do we typically link to issues for fixes?

Reply:
Not often, unless there was significant attention on an issue.   In this case, I believe it was something we noticed internally and investigated, so there's no formal tracking issue associated that we could link to.